content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Groups-based authorization and authorization for list-typed claims
+description: Tutorial on how to configure the groups-base authorization and configure the authorization of list-typed claims in Istio.
+weight: 10
+keywords: [security,authorization]
+---
+
+This tutorial walks you through examples to configure the groups-base
+authorization and the authorization of list-typed claims in Istio.
+
+## Before you begin
+
+* Read the [authorization](/docs/concepts/security/#authorization) concept
+and go through the guide on how to
+[configure Istio authorization](/docs/tasks/security/role-based-access-control).
+
+* Read the Istio
+[authentication policy](/docs/concepts/security/#authentication-policies)
+and the related
+[mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication)
+concepts.
+
+* Create a Kubernetes cluster with Istio installed and mutual TLS enabled.
+To fulfill this prerequisite you can follow the Kubernetes
+[installation instructions](/docs/setup/kubernetes/quick-start/#installation-steps).
+
+## Setup the required namespace and services
+
+This tutorial runs in a new namespace called `rbac-groups-test-ns`,
+with two services, `httpbin` and `sleep`, both running with an Envoy sidecar
+proxy. The following command sets an environmental variable to store the
+name of the namespace, creates the namespace, and starts the two services.
+Before running the following command, you need to enter the directory
+containing the Istio installation files.
+
+1.  Set the value of the `NS` environmental variable to `rbac-listclaim-test-ns`:
+
+    {{< text bash >}}
+    $ export NS=rbac-groups-test-ns
+    {{< /text >}}
+
+1.  Create the namespace for this tutorial:
+
+    {{< text bash >}}
+    $ kubectl create ns $NS
+    {{< /text >}}
+
+1.  Create the `httpbin` and `sleep` services and deployments:
+
+    {{< text bash >}}
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n $NS
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n $NS
+    {{< /text >}}
+
+1.  To verify that `httpbin` and `sleep` services are running and `sleep` is able to
+    reach `httpbin`, run the following curl command:
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n"
+    {{< /text >}}
+
+    When the command succeeds, it returns the HTTP code 200.
+
+1.  Get the existing authentication policies of the newly created namespace:
+
+    {{< text bash >}}
+    $ kubectl get policies.authentication.istio.io -n $NS
```

Comment:
Need to check meshpolicies too. 

kubectl get meshpolicies.authentication.istio.io -n $NS

Reply:
Done.

Reply:
Another thought. Maybe instead of checking whether a resource exists, you simply delete all of them. This way, you do not need to check the resources one by one. See example here https://istio.io/docs/tasks/security/role-based-access-control/#before-you-begin.

Reply:
Done: delete all resources instead of checking resources exist.